src/sentry/utils/pytest/selenium.py

```diff
@@ -453,7 +457,16 @@ def pytest_configure(config):
 
 @TimedRetryPolicy.wrap(timeout=15, exceptions=(WebDriverException,), log_original_error=True)
 def start_chrome(**chrome_args):
-    return webdriver.Chrome(**chrome_args)
+    try:
+        return webdriver.Chrome(**chrome_args)
+    except SessionNotCreatedException as e:
+        if "This version of ChromeDriver only supports Chrome version" in e.msg:
+            raise Exception(
+                """ChromeDriver version does not match Chrome version, update ChromeDriver (e.g. if you use `homebrew`):
+
+    brew cask upgrade chromedriver
```

Comment:
I tried the command and it's `brew upgrade --cask chromedriver`

We could potential warn the developer via direnv:
```
❯ chromedriver --version
ChromeDriver 90.0.4430.24 (4c6d850f087da467d926e8eddb76550aed655991-refs/branch-heads/4430@{#429})

❯ /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --version
Google Chrome 90.0.4430.93
```

Reply:
Ah oops, I thought I got that command from `develop.sentry` but now I can't find it 👀 

re: direnv, I'd prefer not because this only applies when running acceptance tests locally, but it could be something we do if this pops up more.